src/services/edusharing/logic/connector.js

```diff
@@ -0,0 +1,167 @@
+const request = require('request-promise-native');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to resolve path to module 'request-promise-native'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4791331)